api_calls/reference/bookings/bookings_read-bookings_write-scope-response.json

```diff
@@ -29,6 +29,7 @@
         "bookings_taxes": [],
         "bookings_tags": [1, 2],
         "booking_comments": [],
+				"conversations": [11],
```

Comment:
Accidental linting issue? And below as well

Reply:
Sigh. The file is created with spaces, instead of tabs. Git is operating on 4 spaces per tab, while vscode is on 2. They are matching in real life, but not in the file format itself.